tests/unit/wmg_processing/test_load_corpus.py

```diff
@@ -121,40 +122,62 @@ def test_expression_matrix_matches_global_index(self):
     def test_raw_expression_matrix_normalized_by_rankit(self):
         pass
 
-    @unittest.skip("removed corpus fixture")
-    @patch("backend.wmg.data.cube_pipeline.extract.copy_datasets_to_instance")
-    @patch("backend.wmg.data.cube_pipeline.extract.get_dataset_s3_uris")
-    def test_corpus_creation_works_as_expected(self, mock_get_uris, mock_copy):
-        load_data_and_create_cube(self.path_to_datasets, self.corpus_name, self.tmp_dir)
-
-        # check obs
-        with tiledb.open(f"{self.corpus_path}/obs", "r") as obs:
-            actual_obs_df = obs.df[:]
-        with tiledb.open(self.fixture_file_path("fixtures/small-corpus/obs"), "r") as obs:
-            expected_obs_df = obs.df[:]
-
-        self.assertTrue(
-            expected_obs_df.development_stage_ontology_term_id.equals(actual_obs_df.development_stage_ontology_term_id)
-        )
-        self.assertTrue(expected_obs_df.obs_idx.equals(actual_obs_df.obs_idx))
-        self.assertTrue(expected_obs_df.dataset_id.equals(actual_obs_df.dataset_id))
-        self.assertTrue(expected_obs_df.dataset_local_cell_id.equals(actual_obs_df.dataset_local_cell_id))
-
-        # check vars
-        with tiledb.open(f"{self.corpus_path}/var", "r") as var:
-            actual_var_df = var.df[:]
-        with tiledb.open(self.fixture_file_path("fixtures/small-corpus/var"), "r") as var:
-            expected_var_df = var.df[:]
-
-        self.assertTrue(expected_var_df.equals(actual_var_df))
-
-        # check expression matrix
-        with tiledb.open(f"{self.corpus_path}/X", "r") as x:
-            actual_x_df = x.df[:]
-        with tiledb.open(self.fixture_file_path("fixtures/small-corpus/X"), "r") as x:
-            expected_x_df = x.df[:]
-
-        self.assertTrue(expected_x_df.equals(actual_x_df))
+    @patch("backend.corpus_asset_pipelines.integrated_corpus.transform.GENE_EXPRESSION_COUNT_MIN_THRESHOLD", 1)
+    @patch("backend.corpus_asset_pipelines.integrated_corpus.job.tiledb.consolidate", new=Mock())  # Slow
+    @patch("backend.corpus_asset_pipelines.integrated_corpus.job.tiledb.vacuum", new=Mock())  # Slow
+    @patch("backend.wmg.data.cube_pipeline.upload_artifacts_to_s3", new=Mock())  # Don't upload the cube.
+    @patch("backend.wmg.data.cube_pipeline.extract.get_dataset_s3_uris", new=Mock(return_value={}))
+    def test_corpus_creation_works_as_expected(self):
```

Comment:
technically, this is testing _snapshot_ creation; the corpus is the input, as a test fixture; rename?